cli/src/auth/oidc.rs

```diff
@@ -261,3 +280,10 @@ pub struct UserInfo {
     pub preferred_username: Option<String>,
     pub email_verified: Option<bool>,
 }
+
+/// Keycloak error response.
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct ResponseError {
+    error: String,
+    error_description: String,
```

Comment:
In some api tests I wrote recently, the generic JSON error looked like
```json
{
  "error": {
    "code": 400, 
    "reason": "Bad Request",
    "description": "The request could not be understood by the server due to malformed syntax."
    }
}
```
which would fail deserialization here. Doesn't actually change the behavior since you are specifically looking for a keycloack code anyway, but something to be aware of.

Reply:
That is very unfortunate, but yeah this is why I only *attempt* to deserialize the error and still print the entire body if it could not be done. I think going without the field is probably the best idea for now since clearly we don't need it at the moment.